### PR TITLE
Create Hadronizer_TuneCP5Up_13TeV_powhegEmissionVeto_3p_LHE_pythia8_F…

### PR DESCRIPTION
**Pull request by jieunyoo**
…ilteredHiggs_pT_cff_.py

add fragment to be used for filter on high pT Higgs

### CODE REVIEW

File: Hadronizer_TuneCP5Up_13TeV_powhegEmissionVeto_3p_LHE_pythia8_FilteredHiggs_pT_cff_.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#this is based on: https://raw.githubusercontent.com/jieunyoo/genproductions/master/genfragments/ThirteenTeV/Hadronizer/Hadronizer_TuneCP5Up_13TeV_powhegEmissionVeto_3p_LHE_pythia8_cff.py
+#it just adds a new filter at the end
+
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5TuneUpSettings_cfi import *
+from Configuration.Generator.Pythia8PowhegEmissionVetoSettings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
```
Comment (menglu21) [inline Hadronizer_TuneCP5Up_13TeV_powhegEmissionVeto_3p_LHE_pythia8_FilteredHiggs_pT_cff_.py:10]: this line should be the concurrent one.

Comment (jieunyoo) [reply Hadronizer_TuneCP5Up_13TeV_powhegEmissionVeto_3p_LHE_pythia8_FilteredHiggs_pT_cff_.py:10]: Does something need to be changed?

**Comment by jieunyoo** [comment]
Hi. Can this be merged?

**Comment by Saptaparna** [comment]
No, it cannot be merged yet. You have to change `Pythia8HadronizerFilter` to `Pythia8ConcurrentHadronizerFilter`

**Comment by jieunyoo** [comment]
Done, thanks!

**Comment by Saptaparna** [comment]
Did you already make the change here?

**Comment by jieunyoo** [comment]
Ok I put this in the wrong folder. I will close this request and put it in the right folder (Hadronizer)

**Pull request closed without merging**